aws_lambda_builders/__init__.py

```diff
@@ -2,4 +2,4 @@
 AWS Lambda Builder Library
 """
 __version__ = '0.0.5'
-RPC_PROTOCOL_VERSION = "0.1"
+RPC_PROTOCOL_VERSION = "0.2"
```

Comment:
Also, there is implementation to compares versions and returns the necessary error. You should implement it. It will be good to implement it within the __main__.py so only the CLI interface is affected